exporters/zipkin/model_test.go

```diff
@@ -809,7 +973,40 @@ func TestTagsTransformation(t *testing.T) {
 			want: nil,
 		},
 		{
-			name: "statusCode",
+			name: "statusCode UNSET",
+			data: tracetest.SpanStub{
+				Attributes: []attribute.KeyValue{
+					attribute.String("key", keyValue),
+				},
+				Status: tracesdk.Status{
+					Code:        codes.Unset,
+					Description: "",
+				},
+			},
+			want: map[string]string{
+				"key": keyValue,
+			},
+		},
+		{
+			name: "statusCode OK",
+			data: tracetest.SpanStub{
+				Attributes: []attribute.KeyValue{
+					attribute.String("key", keyValue),
+					attribute.Bool("ok", true),
+				},
+				Status: tracesdk.Status{
+					Code:        codes.Ok,
+					Description: "",
+				},
+			},
+			want: map[string]string{
+				"key":              keyValue,
+				"ok":               "true",
+				"otel.status_code": strings.ToUpper(codes.Ok.String()),
```

Comment:
Although I understand the specification for trace is stable now, still are there any chances of changing `codes.Ok.String()` in the future? That's the reason I use upper case method instead of using `"OK"` and `"ERROR"` string.

Reply:
The specification is [stable](https://github.com/open-telemetry/opentelemetry-specification/blob/5edc08b671db33a57bbb4901593c2bbf71c89fa9/specification/trace/sdk_exporters/zipkin.md) for the zipkin exporter and it requires these status to be `OK` and `ERROR` respectively. If the values change to something other than this we would not comply with the specification. Therefore, defined the tests to test this exact behavior.